### PR TITLE
Enhancement engine py version api 0 8 x

### PR DESCRIPTION
**Pull request by BenjamenMeyer**
Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [x] Have the PR Validation Tests been run? As appropriate
- [ ] This is a documentation change only

Issues:
- #349 

Purpose:
- Backport #578 to 0.8.x

### CODE REVIEW

**Review by stephengtuggy — APPROVED** [review]
LGTM!

**Review by Loki1950 — APPROVED** [review]

File: engine/CMakeLists.txt [hunk 1]
```diff
@@ -49,6 +49,12 @@ ELSE ()
 ENDIF ()
 
 
+# API Version for Game Assets
+# This is an incrementing number similar to the Google Android API Version
+# allowing us to differentiate our Assets API across multiple versions.
+# If a release is missing this value, then version `1` can be assumed.
+SET(VEGASTRIKE_ASSETS_API_VERSION "1")
```
Comment (stephengtuggy) [inline engine/CMakeLists.txt:56]: This line ended up in here twice. :-) Specifically on the 0.8.x branch.

Comment (Loki1950) [reply engine/CMakeLists.txt:56]: Darn missed that good catch @stephengtuggy .

Comment (BenjamenMeyer) [reply engine/CMakeLists.txt:56]: Probably from a mistake during cherry-picking.

Comment (BenjamenMeyer) [reply engine/CMakeLists.txt:56]: @stephengtuggy  filed #616 to fix this.

**Pull request merged**